src/include/duckdb/planner/bound_result_modifier.hpp

```diff
@@ -38,11 +40,14 @@ class BoundLimitModifier : public BoundResultModifier {
 public:
 	BoundLimitModifier() : BoundResultModifier(ResultModifierType::LIMIT_MODIFIER) {
 	}
-
-	//! LIMIT count
-	int64_t limit = -1;
+	//! LIMIT
+	int64_t limit_val = NumericLimits<int64_t>::Maximum();
 	//! OFFSET
-	int64_t offset = -1;
+	int64_t offset_val = 0;
+	//! Expression in case limit is not constant
+	unique_ptr<Expression> limit{};
```

Comment:
Please remove the initializers; unique pointers are automatically initialized to null as part of the constructor.